framework/include/utils/FillBetweenPointVectorsTools.h

```diff
@@ -245,4 +245,36 @@ bool isBoundaryOpenSingleSegment(ReplicatedMesh & mesh,
  * @return whether the boundary is the external boundary of the given mesh
  */
 bool isExternalBoundary(ReplicatedMesh & mesh, const boundary_id_type bid);
+
+/**
+ * Creates an QUAD4 element that can be extruded in (0 0 1) direction.
+ * @param elem pointer of the element to be created
+ * @param nd_0 pointer of element's first node
+ * @param nd_1 pointer of element's second node
+ * @param nd_2 pointer of element's third node
+ * @param nd_3 pointer of element's fourth node
+ * @param transition_layer_id id of the subdomain that this element belongs to
+ * @return whether the element is flipped to ensure (0 0 1) extrudability
```

Comment:
so are they not (0 0 -1) extrudable then?

Reply:
In the current version of `FancyExtruderGenerator`, if an element can be extruded in (0 0 1) direction, it cannot be extruded in (0 0 -1) direction (negative Jacobian/volume due to the wrong node order)

Reply:
is there an issue about this?

Reply:
> is there an issue about this?

Thanks for your review. I will create an issue for this later.